src/components/biggive-form-field-select/biggive-form-field-select.scss

```
@@ -4,9 +4,25 @@
   display: contents;
 }
 
+.prompt {
+  position: absolute;
+  z-index: 2;
+  font-size: small;
+  top: -1em;
+  background-color: $colour-white;
+  left: 4em;
+  padding-left: 1em;
+  padding-right: 1em;
+  color: $colour-primary-blue;
+}
+.prompt.grey {
+  background-color: $colour-grey-background;
+}
+
 .dropdown {
   @include standard-font();
   position: relative;
+  margin-top: 1.0em;
```

```suggestion
  margin-top: 1em;
```
I would go with this just for search-ability if we're ever debugging the layout